inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl

```diff
@@ -17,65 +17,172 @@
-      </div>
-      <div class="modal-body">
-        {% for author in record.authors %}
-          <small class="text-left">{{ render_author_names(author) }}</small>
-          <small>{{ sep() }}</small>
-        {% endfor %}
-      </div>
-      <div class="modal-footer">
+    <div class="modal fade" id="authors_{{ record['control_number'] }}" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            <h4 class="modal-title" id="myModalLabel">Authors</h4>
+          </div>
+          <div class="modal-body">
+            {% for author in record.authors %}
+              <small class="text-left" >
+              <a href="{{ url_for('search.search', p='author:"' + author.get('full_name') + '"') }}">
+               {{ author.get('full_name') }}
+              </a>
+               {% if author.get('affiliation') %}
+                {% if author.get('affiliation') | is_list %} 
+                  <a href="{{ url_for('search.search', p='"' + author.get('affiliation')[0] + '"' + "&cc=Institutions") }}">
+                    ({{ author.get('affiliation')[0] }})
+                  </a>
+                {% else %}
+                  <a href="{{ url_for('search.search', p='"' + author.get('affiliation') + '"' + "&cc=Institutions") }}">
+                    ({{ author.get('affiliation') }}) 
+                  </a>
+                {% endif %} 
+              {% endif %}
+              </small>
+              <small>{{ sep() }}</small>
+            {% endfor %}
+          </div>
+          <div class="modal-footer">
+          </div>
+        </div>
       </div>
     </div>
-  </div>
-</div>
-  {% endif %}
   {% endif %}
 {% endmacro %}
 
-{% macro record_arxiv() %}
-  {% if record.get('arxiv_eprints') %}
-    {% if record.get('arxiv_eprints') | is_list() %}
-      {% set filtered_arxiv = record.get('arxiv_eprints') %}
-      {% for i in filtered_arxiv %}
-        <b>e-Print:</b>
-          <a href="http://arxiv.org/abs/{{ i.get('value') }}">{{ i.get('value') }}</a>
-        {% if i.get('categories') %}
-          &nbsp;<b>[{{  i.get('categories')|join(',')  }}]</b>
+{% macro record_abstract(is_brief) %}
+  {% if is_brief %}
+    {% set number_of_words = 30 %}
+  {% else %}
+    {% set number_of_words = 100 %}
+  {% endif %}
+  {% set abstract_exists = [] %}
+  {% if record.get('abstract') and record.get('abstract') | is_list %}
+    {% if record.get('abstract')| count > 1%}
+      {% for abstract in record.get('abstract') %}
+        {% if abstract.get('value') and abstract.get('source') != 'arXiv' %}
+          {% do abstract_exists.append(1) %}
+        {% endif %}
+      {% endfor %}
+      {% for abstract in record.get('abstract') %}
```

Comment:
The logic of what abstract to display when there are multiple should follow the same rules as in the current inspire. ( see https://github.com/inspirehep/inspire/blob/master/bibformat/format_elements/bfe_INSPIRE_abstract.py#L75-L90 )

* If there are other abstracts apart from the arXiv one, those take precedence. (For example https://inspirehep.net/record/1393642)

I am not sure about the usefulness of showing a second abstract, so for the moment I would show only one.